test/uk/gov/hmrc/cardpaymentfrontend/controllers/PaymentFailedControllerSpec.scala

```diff
@@ -156,6 +156,27 @@ class PaymentFailedControllerSpec extends ItSpec {
 
     }
 
+    "When Open Banking is selected" - {
```

Comment:
are any of the error scenarios covered? i.e. if no radio is selected but the form is submitted.